chains/evm/contracts/interfaces/ICrossChainVerifierV1.sol

```diff
@@ -32,22 +32,23 @@ interface ICrossChainVerifierV1 is IERC165 {
     bytes memory ccvData
   ) external;
 
-  /// @notice Quotes the fee for a CCIP message to a destination chain.
-  /// @dev This takes EVM2AnyMessage (instead of MessageV1) because
-  /// the router client API that user contracts interact with (IRouterClient.getFee)
-  /// exposes EVM2AnyMessage. The on-ramp can translate to MessageV1 internally
-  /// where required (e.g., verifier hooks), but using EVM2AnyMessage here keeps the
-  /// interface aligned with what clients construct and pass to the router.
-  /// @param originalCaller The original caller of getFee.
+  /// @notice Quotes the fee, including gas and calldata bytes, for a CCIP message to a destination chain.
+  /// @dev This takes EVM2AnyMessage (instead of MessageV1) because the router client API that user contracts interact
+  /// with exposes EVM2AnyMessage. The onRamp can translate to MessageV1 internally where required (e.g., verifier or
+  /// executor hooks), but using EVM2AnyMessage here keeps the interface aligned with what clients construct and pass to
+  /// the router.
+  /// @param originalCaller The original caller of getFee, this is required to support proxy patterns.
   /// @param destChainSelector The destination chain selector of the message.
   /// @param message The message to be sent.
-  /// @param extraArgs Opaque extra args that can be used by the fee quoter
+  /// @param extraArgs Opaque extra args that can be used by the fee quoter,
+  /// @param finalityConfig Finality configuration.
   function getFee(
     address originalCaller,
     uint64 destChainSelector,
     Client.EVM2AnyMessage memory message,
-    bytes memory extraArgs
-  ) external view returns (uint256);
+    bytes memory extraArgs,
+    uint16 finalityConfig
```

Comment:
Is the naming of `finalityConfig` intentionally opaque? So different verifiers can use different units (i.e. blocks, time) to represent finality? Just seems unusual that a number would be given the label "config".

Reply:
Yeah can rename it, was originally not just block depth but now it is. Kinda, if you ignore 0.

Not relayed to this PR though, so can we do that in another one?